### PR TITLE
CF-45 Date Type Change and Comment Issues

### PR DESCRIPTION
**Pull request by erencanuysal**
Proposed changes
----------------
I fixed date types in both the creation and visualization of posts.
I also fixed the issues about newly created comments and replied comments

image
image
image

Types of changes
----------------

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

Checklist
---------

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Further comments
----------------

(Write your answer here.)

### CODE REVIEW

File: Columbus/frontend/src/components/Post/Post.js [hunk 1]
```diff
@@ -121,19 +121,23 @@ export default function Post(props) {
...
-      if(props.post.time_end[0].month)
-        setStoryDate2(props.post.time_end[0].month+"."+props.post.time_end[0].year)
-      if(props.post.time_end[0].day)
-        setStoryDate2(props.post.time_end[0].day+"."+props.post.time_end[0].month+"."+props.post.time_end[0].year)
-      }
-    }
+    else if(props.post.time_start[0].type=="decade")
+      setStoryDate1(props.post.time_start[0].date+"0s")
+    if(props.post.time_end.length >0 ){
+      console.log(props.post.time_end)
```
Comment (onurcanavci) [inline Columbus/frontend/src/components/Post/Post.js:127]: Burada console unutulmuş

Comment (erencanuysal) [reply Columbus/frontend/src/components/Post/Post.js]: Teşekkürler, düzelttim

**Review by onurcanavci — APPROVED** [review]
Eline sağlık, yorucu bir pr olsa gerek :)

**Comment by erencanuysal** [comment]
> Eline sağlık, yorucu bir pr olsa gerek :)

Teşekkürler, cidden öyleydi :)

**Pull request merged**